pkg/gui/keybindings.go

```diff
@@ -318,299 +601,299 @@ func (gui *Gui) GetInitialKeybindings() []*Binding {
 		},
 		{
 			ViewName:    "files",
-			Key:         'f',
+			Key:         gui.getKey("files.fetch"),
 			Modifier:    gocui.ModNone,
 			Handler:     gui.handleGitFetch,
 			Description: gui.Tr.SLocalize("fetch"),
 		},
 		{
 			ViewName:    "files",
-			Key:         'X',
+			Key:         gui.getKey("files.executeCustomCommand"),
```

Comment:
because this PR will lock-in what our keybindings are in a way that will cause issues for users when we make changes, I'm going to suggest a few minor tweaks to the existing keybindings which make a bit more sense (but that perhaps weren't yet changed due to some other problems we can address later).

In this case I think 'X' should execute a custom command no matter which panel we're in. Unfortunately we currently kick you back to the files panel when we come back from the subprocess which is why I had this scoped to just the files panel, however I think it makes sense for this to just be global.

Reply:
Yes, I agree.